geode-core/src/main/java/org/apache/geode/internal/cache/AbstractRegion.java

```diff
@@ -741,8 +741,8 @@ public boolean isPdxTypesRegion() {
         // This is for all regions except pdx Region
         if (!isPdxTypesRegion) {
           // Make sure we are distributing to only those senders whose id
-          // is available on this region
-          if (allGatewaySenderIds.contains(sender.getId())) {
+          // is available on this region and whose state is running
+          if (allGatewaySenderIds.contains(sender.getId()) && sender.isRunning()) {
             allRemoteDSIds.add(sender.getRemoteDSId());
           }
         } else { // this else is for PDX region
```

Comment:
Do we also need a check if sender is running for PDX region

Reply:
I tested it and it works without checking if sender is running. All entries will be delivered to all remote sites for pdx regions.